hazelcast/src/main/java/com/hazelcast/core/MemberLeftException.java

```diff
@@ -57,13 +58,14 @@ private void writeObject(ObjectOutputStream out) throws IOException {
         out.defaultWriteObject();
```

Comment:
Are we sure this exception isn't picked up by the client? If so, there could be a compatibility issue.